etl/harmonize.py

```diff
@@ -22,7 +22,10 @@
 @click.argument("data_file")
 @click.argument("column")
 @click.argument("output_file")
-def harmonize(data_file: str, column: str, output_file: str) -> None:
+@click.argument("institution", required=False)
```

Comment:
Have you considered using `@click.option` instead? Citing [click.argument](https://click.palletsprojects.com/en/8.1.x/api/#click.argument)

> This is to follow the general convention of Unix tools of using arguments for only the most necessary things, and to document them in the command help text by referring to them by name.

Using `click.option` would also let you add `help=` clarifying what does it do.

Reply:
I think I prefer the current option. If you see the help message, it looks like:
```
Usage: harmonize [OPTIONS] DATA_FILE COLUMN OUTPUT_FILE [INSTITUTION]
...

Options:
  --help  Show this message and exit.
``` 
which is clearer than when using `click.option("-i", "--institution")`:
```
Usage: harmonize [OPTIONS] DATA_FILE COLUMN OUTPUT_FILE
...
Options:
  -i TEXT  Optionally add institution name
  --help   Show this message and exit.
```